.github/workflows/scripts/.pinot_linter.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -x
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Java version
+java -version
+
+# Check network
+ifconfig
+netstat -i
+
+# mvn clean install -DskipTests -T 16 || exit 1
+
+mvn rat:check
+mvn checkstyle:check
+mvn spotless:check
+mvn enforcer:enforce
+
+mvn clean > /dev/null
```

Comment:
no need to clean since we didn't build the project

Reply:
```suggestion
```